src/java/org/apache/cassandra/locator/DynamicEndpointSnitch.java

```diff
@@ -378,6 +381,34 @@ public double getSeverity()
         return getSeverity(FBUtilities.getBroadcastAddressAndPort());
     }
 
+    @Override
+    public void setQuantile(double quantile)
+    {
+        if (quantile < 0.0 || quantile > 1.0 || Double.isNaN(quantile))
+        {
+            throw new IllegalArgumentException(quantile + " is not in [0..1]");
+        }
+        replicaLatencyQuantile = quantile;
```

Comment:
shouldn't we recalculate scores after changing this param?

Reply:
We could, but SRP says no.
The change gets effective after the next score update which is 50ms on avg. Why care?